config.json

```diff
@@ -1031,6 +1031,11 @@
         "Lists"
       ]
     },
+      {
```

Comment:
This object needs some more properties:
- `uuid`: sorry, I don't know where these ids are coming. What about copying the `uuid` from the Java track?
- `core`: it should be set to `false`, because we have already decided all *core* exercises
- `unlocked_by`: it should point to a core exercise. I'd suggest `leap`, `space-age` or `prime-factors`. Just choose one.

I'd also add some more topics, `Mathematics`, `Integers` could be some of them. You'll find here a list of [available topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt)